### PR TITLE
Remove unused code, round 1

### PR DESCRIPTION
**Pull request by jeherve**
I started reviewing files and removing unused code here and there in the plugin. I'll continue to do so over the next few weeks, but I thought I'd already submitted what I have done so far, to avoid having only one huge Pull Request, that would be harder to review.

### CODE REVIEW

**Comment by samhotchkiss** [comment]
You're amazing.

File: class.jetpack-user-agent.php [hunk 1]
```diff
@@ -1193,45 +1193,47 @@ static function get_blackbeberry_OS_version() {
...
 					$version = $matches[1];
 					$version_num = explode( '.', $version );
-					if( is_array( $version_num ) === false || count( $version_num ) <= 1 )
-						return 'blackberry-6'; //not a BB device that match our rule.
-					else
-					return 'blackberry-'.$version_num[0];
+					if ( is_array( $version_num ) === false || count( $version_num ) <= 1 ) {
+						return 'blackberry-6'; // not a BB device that match our rule.
+					} else {
+						return 'blackberry-'.$version_num[0];
```
Comment (zinigor) [inline class.jetpack-user-agent.php]: This is awesome, but needs a space in concatenation. Sorry for the nitpick :)

**Comment by zinigor** [comment]
Read the changes, tested affected functionality, everything looks good!

File: class.jetpack-user-agent.php (hunk 1 above)
Comment (jeherve) [inline class.jetpack-user-agent.php]: Fixed!

**Pull request merged**